backend/ideas/utility/idea.py

```diff
@@ -149,3 +151,20 @@ def get_idea_short_description(idea: Idea, language_code: str) -> str:
         )
 
     return idea.short_description
+
+
+def get_number_of_idea_comments(idea: Idea):
+    return IdeaComment.objects.filter(
+        idea=idea,
+        is_abusive=False,
+        deleted_at__isnull=True,
+        parent_comment=None,
+    ).count()
+
+
+def get_number_of_idea_participants(idea: Idea):
+    chats = MessageParticipants.objects.filter(name=idea.name)
+    for chat in chats:
+        user_count = Participant.objects.filter(chat=chat, is_active=True).count()
```

Comment:
Can you update this query to something like this,

```python
chat_user_count = Participant.objects.filter(chat__name=idea.name, is_active=True).count()
```

This makes one less query to our database and avoids a for loop you have in your code.

Reply:
Thanks for the suggestion, changed it :)